docs/docs/typechecking-with-proptypes.md

```diff
@@ -8,7 +8,7 @@ redirect_from:
 
 > Note:
 >
-> `React.PropTypes` has moved into a different package since React v15.5. Please use [the `prop-types` library instead](https://www.npmjs.com/package/prop-types).
+> `React.PropTypes` was moved into a different package on React v15.5. Please use [the `prop-types` library instead](https://www.npmjs.com/package/prop-types).
```

Comment:
Sorry.  For a second there I was imagining "React v15.5" was a date.  So, for example, "React.PropTypes" was moved into a different package on Dec, 13 2016" would have been correct.  

Looking at it again, I think we could incorporate your `as of` and say `React.PropTypes is in a different package as of React v15.5`  or `as of React v15.5, React.PropTypes is in a different package` or even `since React v15.5, React.PropTypes is in a different package.`    But 'has moved' and 'since' don't work together unless it has been moving to the new package this entire time.